java8/src/main/java/com/dslplatform/json/processor/ConverterTemplate.java

```diff
@@ -1073,7 +1073,10 @@ private void processPropertyValue(
 				code.append(alignment).append("\t\t_").append(attr.name).append("_ = ");
 			}
 			List<String> types = attr.collectionContent(context.typeSupport, context.structs);
-			if (attr.converter != null) {
+			if (useInstance && attr.converter != null && !attr.converter.binder.equals("") && attr.field != null) { //todo
+				code.append(attr.converter.fullName).append(".").append(attr.converter.binder);
+				code.append(".bind(reader, instance.").append(attr.field.getSimpleName()).append(")");

```

Comment:
I assume we should do this binds only when properties are non-null. Otherwise we can do just read.